.github/workflows/gradle-build.yml

```diff
@@ -17,4 +17,4 @@ jobs:
 
       - run: ./gradlew build
         env:
-          DEVELOCITY_ACCESS_KEY: ${{ secrets.GE_SOLUTIONS_ACCESS_TOKEN }}
+          DEVELOCITY_ACCESS_KEY: ${{ secrets.DV_SOLUTIONS_ACCESS_KEY }}
```

Comment:
Not necessarily in this PR, but we should update the workflows to use `setup-gradle@v4` and generate [short-lived access tokens](https://github.com/gradle/actions/blob/main/docs/setup-gradle.md#managing-develocity-access-keys).

Reply:
Created a PR: #12